### ISSUE TITLE
Incorrect property 'jmUserPropertiesFiles' in documentation

### ISSUE THREAD

**Issue by enedzvetsky** [jmeter-gradle-plugin/jmeter-gradle-plugin#75]
Page https://github.com/jmeter-gradle-plugin/jmeter-gradle-plugin/wiki/Getting-Started contains incorrect example in 'Simple Configuration' section. Property 'jmUserPropertiesFiles' doesn't exist in JMPluginExtension.

**Comment by rachid-o** [comment]
Correct, I had the same issues when just starting with this plugin. 
I changed the example to:
`jmSystemPropertiesFiles= [file("src/test/jmeter/jmeter.properties")] //to add additional system properties`
